exercises/run-length-encoding/src/RunLength.hs

```diff
@@ -0,0 +1,9 @@
+module RunLength (decode, encode) where
+
+import Data.Char (isDigit)
```

Comment:
There are many ways to solve this problem without using `isDigit`, and it is not need for the type signature.

This import would direct people to look for a specific solution, reducing diversity. I think that it would be removed.